apps/frontend/src/app/containers/signUp/SignUp.tsx

```diff
@@ -0,0 +1,57 @@
+import { IUser } from '@innbyggerpanelet/api-interfaces';
+import { Button, Panel } from '@navikt/ds-react';
+import { ChangeEvent, MouseEvent, ReactElement, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { UserEditCriterias } from '.';
+import { createUser } from '../../api/mutations/mutateUser';
+import { UserContactInfoForm } from '../../components/user';
+
+import style from './SignUp.module.scss';
+
+const defaultUser: IUser = {
+    id: 0,
+    name: '',
+    email: '',
+    phone: '',
+    latestUpdate: Date.now().toString(),
+    criterias: []
+};
+
+export const SignUp = (): ReactElement => {
```

Comment:
Rename

Reply:
Seems like the files were duplicated for some reason :P